### PR TITLE
Upgrade to Symfony 6

### PR DESCRIPTION
**Pull request by PGBastien**
This PR allow to upgrade Symfony from `5.4` to `6.4`:

- [X] [Upgrade PHP dependencies](https://github.com/najmi9/mercure-symfony-chat/commit/ebbdf02e348e0d1acd5e6fdffb88fc869b673d41)
- [X]  [Upgrade PHP 7.4 to 8.3](https://github.com/najmi9/mercure-symfony-chat/commit/b3274c0cf9ddd5538ad4786fffcd6845585c5b67)
- [X] [Upgrade Symfony 5.4 to 6.4](https://github.com/najmi9/mercure-symfony-chat/commit/4673c722922e2291014d48bce53ec572a129f946)
- [X] Replace PHP annotations by attributes

And a fix on param page undefined.

Actual Symfony LTS = `6.4.2` => https://symfony.com/releases

Source:
- https://symfony.com/doc/6.4/routing.html#creating-routes-as-attributes
- https://symfony.com/doc/6.4/security.html#loading-the-user-the-user-provider
- https://symfony.com/doc/6.4/security/access_control.html
- https://symfony.com/doc/6.4/security/access_denied_handler.html#customizing-all-access-denied-responses
- https://symfony.com/blog/new-in-symfony-5-3-passwordhasher-component
- https://symfony.com/doc/6.4/reference/attributes.html

### CODE REVIEW

File: src/Entity/User.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
  * @ORM\Entity(repositoryClass=UserRepository::class)
  * @UniqueEntity(fields={"email"}, message="There is already an account with this email")
```
Comment (najmi9) [inline src/Entity/User.php]: @PGBastien Can we use php 8 attributes here also instead of annotations??

Comment (PGBastien) [reply src/Entity/User.php]: Hi @najmi9 good idea 👍🏼 
[`7a0b6fa` (#2)](https://github.com/najmi9/mercure-symfony-chat/pull/2/commits/7a0b6faf6f8b75a7ec0ed51c8bd535d387eb7e19)

Comment (najmi9) [reply src/Entity/User.php]: Thanks 🧡.

**Review by najmi9 — APPROVED** [review]
Really great PR, thanks.

**Comment by PGBastien** [comment]
> Really great PR, thanks.

With pleasure 👍🏼

**Pull request merged**